### PR TITLE
transform-inline-environment-variables: include and exclude option

### LINKED ISSUE #513: babel-plugin-transform-inline-environment-variables - Blacklist/whitelist?
I've been using `babel-plugin-node-env-inline` for a while when compiling code that is to be used in Node. I'm getting to the point where I would like to inline other environment variables for dead code elimination while not inlining others for use with `dotenv`. Would it make sense to have an optional blacklist/whitelist to specify which environment variables should be inlined or not?

### PR DESCRIPTION
**Pull request by christophehurpeau**
Fixes #513
Closes #531 

Same as #531 where test fails

### CODE REVIEW

**Comment by boopathi** [comment]
Can you change it to `include` and `exclude`.

**Comment by christophehurpeau** [comment]
@boopathi changed as requested

File: packages/babel-plugin-transform-inline-environment-variables/README.md [hunk 1]
```diff
@@ -48,3 +55,8 @@ require("babel-core").transform("code", {
   plugins: ["transform-inline-environment-variables"]
 });
 ```
+
+## Options
+
++ `include` - Array indicating which inline environment variables you want to inline
```
Comment (vigneshshanmugam) [inline packages/babel-plugin-transform-inline-environment-variables/README.md]: typo `inline -> include`

Comment (christophehurpeau) [inline packages/babel-plugin-transform-inline-environment-variables/README.md]: Thanks

**Comment by boopathi** [comment]
#531 is merged. Thanks.

**Pull request closed without merging**